app/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Werkzeug
+git+https://github.com/rwth-iat/basyx-python-sdk@main
```

Comment:
I'm against using a requirements.txt since it is 
- a) deprecated
- b) our only dependency should be the BaSyx-Python-SDK. (The missing Werkzeug dependency is a bug of the BaSyx-Python-SDK See [eclipse-basyx#306](https://github.com/eclipse-basyx/basyx-python-sdk/issues/306))

So personally, I'd onstall it via the `pip install`

Reply:
`requirements.txt` is deprecated?

Reply:
Okay, I seem to have gotten that wrong. Here's the relevant discussion: https://github.com/pypa/packaging.python.org/issues/685#issuecomment-1321616748

However, my point b) still stands and I think using requirements.txt here is unnecessary.